### PR TITLE
Hopefully adds condiment crate

### PR DESCRIPTION
**Pull request by jughu**
for all your cheffing needs 

:cl: Improvedname
rscadd: Adds the condiment kit to cargo ordering
/:cl:

### CODE REVIEW

**Comment by Fullofskittles** [comment]
I don't really understand why we need a crate like this. You can already order replacement pepper/salt from the normal Food Crate. Everything elsecan be made by blending the right foods from botany with the exception of "borksauce" which...doesn't seem like something you should easily be adding to everything :p

**Comment by jughu** [comment]
sure let me get one bottle of ketchup sauce and grind up like 100 tomatoes

**Comment by jughu** [comment]
and borksauce is auctally msg that adds flavortext that says This tastes amazing!

**Comment by jughu** [comment]
i mean it does have a drawback but thats if you ingest too many

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -877,6 +877,19 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	announce_beacons = list("Kitchen" = list("Kitchen"))
 
 /datum/supply_packs/organic/pizza
```
Comment (FlattestGuitar) [inline code/datums/supplypacks.dm]: The new crate needs to have a new path, this isn't a `pizza` crate

File: code/modules/reagents/oldchem/reagents/reagents_food.dm [hunk 2]
```diff
@@ -86,6 +86,10 @@
 	reagent_state = LIQUID
 	color = "#B31008" // rgb: 179, 16, 8
 
+/datum/reagent/capsaicin/reaction_mob(mob/living/M, method=TOUCH, volume)
+	if(method == INGEST)
+		to_chat(M, "<span class='notice'>Yikes, thats spicy!</span>")
```
Comment (FlattestGuitar) [inline code/modules/reagents/oldchem/reagents/reagents_food.dm]: The word "Yikes" fills me with dread.

**Comment by FlattestGuitar** [comment]
Personally, I don't see anything wrong with this. Chefs rarely make condiments and it's a tiny bit of fluff that adds to the game. Why not?

**Comment by jughu** [comment]
Maybe chefs would make condiments more if it didnt take like hundred tomatoes than the average plant yield

**Comment by IcyV** [comment]
Monosodium is used largely in the creation of bath salts. The only other real ways to get it other than bacon is to empty the entire Changs vending machine and grind every piece down for just a little bit. I think this will just make it too easy to even more rapidly mass produce bath salts a lot of the time.

Also not too horribly big on frostoil being so easily available. That should be something that needs to be worked for since you can do some very unique things with it. The other ways are ice peppers and xenobio, which I think is a fair requirement.

I think those two should be taken out and the hotsauce/ketchup should be added to the normal food crates along with the other little things that already come in those.

As condiments they're harmless fluff. Not so much if someone other than the chef gets their hands on them.

**Comment by jughu** [comment]
what if i still add the msg but at a lower amount maybe 10 since its SPECIAL and SPECIAL is EXPENSIVE i will remove the frostoil then @IcyV

**Comment by jughu** [comment]
i also can low the frostoil to like 10 units too since you need 1 unit to apply the fluff text and its not like a scientist can order like 10 crates of condiments and such without the cargo being suspicous

**Comment by jughu** [comment]
and be honest its not like a chef EVER gets his hand on frost oil

**Comment by jughu** [comment]
from xenobiology that is

**Comment by jughu** [comment]
altough i can do a locked crate with kitchen access i mean if chef plants to do something antaggy with those 50 units he deserves it

**Comment by jughu** [comment]
honestly yeah i think its a better idea adding it now

**Comment by jughu** [comment]
and if somoene goes trough all the trouble to steal it from the chef they honestly deserve it its like stealing a virus crate from a viroglist

**Comment by Plotron** [comment]
Damn it, man.

Just add various condiments and restrict special chems to like 5-10 units (single packet). Especially the frostoil.
The borksauce (if you're right) could be limited to 15-20 units instead.

That chef access lock is quite hilarious, IMO, and as such I don't mind it. :D

**Comment by jughu** [comment]
ONLY FOR THE CHEF ONLY FOR THE CHEF

**Comment by jughu** [comment]
The borksauce is fine if you go trough all the trouble to steal it from the chef just to create shitty bath salts they honestly deserve this you poeple say its like a OP thing even meth is better and easier to make

**Comment by Fox-McCloud** [comment]
Most reagents shouldn't have taste associated with them on ingest. It removes stealth in a big way--generally speaking reagents that do something on ingest do so because they deal damage or it's for comedic effect.

Yes, I know MSG has a "taste", but it's a fluff/joke chem

**Comment by jughu** [comment]
Oke oke i remove that then haha

**Comment by Fox-McCloud** [comment]
I'm kinda inclined to agree that this seem a bit...unnecessary...especially with how easy it is for these condiments to be made if botany just does their damn job.

**Comment by jughu** [comment]
but its annoying to grind like 20 tomatoes to have on full bottle its really tiring

**Comment by jughu** [comment]
it takes atleast 4 plant yields

**Comment by Fox-McCloud** [comment]
Going to close this. IT doesn't seem that popular of an idea, and upon second reconsideration, I recall that the chef's vending machine has kethcup and hot sauce packets in it, both---in addition to a ton of salt and pepper shakers.

So really it's just a crate that has an MSG bottle and frostoil bottle in it--which doesn't really seem necessary, TBH.

**Pull request closed without merging**